mxcube3/routes/loginutils.py

```diff
@@ -93,7 +93,8 @@ def set_operator(sid):
     user = get_user_by_sid(sid)
     user["operator"] = True
 
-    select_proposal(user["loginID"])
+    if mxcube.db_connection.loginType.lower() != 'user':
```

Comment:
This is important for remote usage; to change the data path when operator changes. You would probably need to do something here to handle that.

Reply:
Okey I see, in our case we rely on the select proposal modal in the client, so I would need to implement that... later ok?